src/layer/Tooltip.js

```diff
@@ -143,11 +143,11 @@ export var Tooltip = DivOverlay.extend({
 		    anchor = this._getAnchor();
 
 		if (direction === 'top') {
-			subX = tooltipWidth / 2;
+			subX = tooltipWidth / 2 + anchor.x;
 			subY = tooltipHeight;
 		} else if (direction === 'bottom') {
-			subX = tooltipWidth / 2;
-			subY = 0;
+			subX = tooltipWidth / 2 + anchor.x;
+			subY = anchor.y;
 		} else if (direction === 'center') {
 			subX = tooltipWidth / 2;
```

Comment:
We need to also change this into:

`subX = tooltipWidth / 2 + anchor.x;`

to properly x align the tooltip also for the "center" use case

Reply:
With the above amend this looks good to me for merging into main branch.